study.md

```diff
@@ -21,18 +21,36 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
-```
+1. i dont't understand how callback functions are called back at a later time.
+2.  I don't understand an exampe of a function in an article:
+function example(a,b) {
+	number += a;
+	alert('You have chosen: ' + b);
+};
+what is the point of line 28 since the function just gives back an alert?
+
+3. I am confused about function declaration and expression
+is this a function decleration?:
+function foo() {
+ console.log('hello');
+}; and can be called above where it's defined because it's hoisted?
+
+and is this a function expression?:
+const x = function foo() {
+ console.log('hello');
+}; and it can only be called below it is defined? ```
 
 Are there any nagging questions relating to functions that you have?
 
 ```md
-<!-- your answer here -->
-```
+See the 1st question.```
 
 Thus far how are you feeling about your progress as a developer, please give
 an extensive answer?
 
 ```md
-<!-- your answer here -->
-```
+I have mixed feelings.
+I completly understand GitHub (for now, I'm sure there is more to it than we've been taught)
+I also feel super about the basic of Java that we covered during the 2nd day.
```

Comment:
This get mixed up sometimes, but Java and JavaScript are two different languages and, with the exception of both being C-like, they don't really have anything to do with each other.